tests/unit/ExampleTest.php

```diff
@@ -0,0 +1,22 @@
+<?php 
+class ExampleTest extends \Codeception\Test\Unit
+{
+    /**
+     * @var \UnitTester
+     */
+    protected $tester;
+    
+    protected function _before()
+    {
+    }
+
+    protected function _after()
+    {
+    }
+
+    // tests
+    public function testSomeFeature()
+    {
+
+    }
```

Comment:
ERROR: Function closing brace must go on the next line following the body; found 1 blank lines before brace